W-12655114-connectors-ref-kt (#2259)
mulesoft/docs-connectors: docusign/1.0/modules/ROOT/pages/docusign-connector-reference.adoc

`<docusign:create-v21-accounts-bulk-send-lists-send-by-account-id-bulk-send-list-id>`

> Initiates the bulk send process. It generates a bulk send request based on an existing bulk send list and an envelope or template. Consider using the createBulkSendTestRequest method to test your bulk send list for compatibility with the envelope or template that you want to send first. If the envelopes were successfully queued for asynchronous processing, the response contains a `batchId` that you can use to get the status of the batch. If a failure occurs, the API returns an error message.

Reviewer: ```suggestion
Initiates the bulk send process by generating a bulk send request based on an existing bulk send list and generating an envelope or template. Consider using the createBulkSendTestRequest method to test your bulk send list for compatibility with the envelope or template that you want to send first. If the envelopes are successfully queued for asynchronous processing, the response contains a `batchId` that you can use to get the status of the batch. If a failure occurs, the API returns an error message.
```

Reviewer: I am not sure if my proposed rewording is accurate. Either way, you should remove the "It generates."

For createBulkSendTeset Request, this should be spelled out as it appears in Studio.
